test/unit/three_scale/swagger/specification_test.rb

```diff
@@ -13,6 +13,16 @@ class ThreeScale::Swagger::SpecificationTest < ActiveSupport::TestCase
     assert specification.valid?, 'specification should be valid'
   end
 
+  test "invalid 2.0 spec errors" do
+    specification = fixture_spec('invalid-2.0')
+    assert_not specification.valid?
+    errors = specification.errors.full_messages_for(:base).join(', ')
+    assert errors.include? "The property '#/' contains additional properties [\"invalid\", \"another\"] outside of the schema when none are allowed"
```

Comment:
we have `assert_includes` assertion that will show a more meaningful error message if it fails

Reply:
Yeah, I saw it, but I was kind of misled by the fact that the method definition talks about "collection" containing the object. But it's true that it can still apply here, because that assertion just calls `inlcude?` as well.

But honestly, I am not sure verbosity is good in this case.

If I introduce a change in the string, the error is currently reported as:

```
Minitest::Assertion: Expected false to be truthy.
test/unit/three_scale/swagger/specification_test.rb:20:in `block in <class:SpecificationTest>'
```
so you can easily see what the problem is - obviously, `include?` is not true.


With `assert_includes` the error shows as:

```
Minitest::Assertion: Expected "The property '#/' contains additional properties [\"invalid\", \"another\"] outside of the schema when none are allowed in schema http://swagger.io/v2/schema.json#, The property '#/info' did not contain a required property of 'title' in schema http://swagger.io/v2/schema.json#, The property '#/info' contains additional properties [\"foo\"] outside of the schema when none are allowed in schema http://swagger.io/v2/schema.json#" to include "Theq property '#/' contains additional properties [\"invalid\", \"another\"] outside of the schema when none are allowed".
test/unit/three_scale/swagger/specification_test.rb:20:in `block in <class:SpecificationTest>'
```

This is very long, and hard to follow. Obviously, if you trace back to the line, you can also see what is done there, and understand why it fails, but you would hardly read the message :grimacing: